### PR TITLE
[RFR] Add component autoloading

### PR DESCRIPTION
**Pull request by manuquentin**

### CODE REVIEW

File: app/ReactAdmin.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
-import CreateView from './View/Create';
-import EditView from './View/Edit';
-import DeleteView from './View/Delete';
+import './AdminBootstrap';
+import './View/Dashboard';
+import './View/List';
+import './View/Show';
+import './View/Create';
+import './View/Edit';
+import './View/Delete';
```
Comment (jeromemacias) [inline app/ReactAdmin.js]: Do you really need to use autoloader internally ?

**Comment by jeromemacias** [comment]
Tests failed.

File: app/autoloader.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+function register(name, value) {
+    global.__defineGetter__(name, () => value);
```
Comment (fzaninotto) [inline app/autoloader.js]: `__defineGetter__` is non-standard AND deprecated:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/__defineGetter__

We need to find an alternative.

Comment (manuquentin) [inline app/autoloader.js]: Fixed in #47

**Pull request merged**